### ISSUE TITLE
Ghostcommander just get Betas and throws a warning after update search

### ISSUE THREAD

**Issue by topcaser** [ImranR98/Obtainium#1537]
**Describe the bug**

I once included ghostcommander via this link: https://sourceforge.net/projects/ghostcommander/files

Now I get an error message if I let search for updates: No matching version found for Ghost commander

The installed version is according obtainium: it just says Beta

**Please complete the following information:**
 - Device:  pixels 8 pro 
 - OS:  Graphenos 
 - Obtainium Version:  1.1.3

**Comment by ImranR98** [comment]
The SourceForge code was removed a couple of versions ago because it was redundant - the HTML source can be used for SF and is a lot more flexible.

In this case, you should remove the app from Obtainium and re-add it with one of these 2 URLs (depending on need):
- https://sourceforge.net/projects/ghostcommander/files/Releases/
- https://sourceforge.net/projects/ghostcommander/files/Beta/

Your "APK link filter" should be set to `Commander [0-9]` assuming you want the main app, not any of the plugins (which would need to be added separately anyways).

Your "version extraction" filter can be set to `Commander (.+)\.apk` with a "match group" of `1`.

It's also important that you delete the default headers setting.

**Comment by ImranR98** [comment]
Actually, seems like the settings above should work, but don't. Found a bug that will be fixes in the next release - the above settings should work after that.

**Comment by ImranR98** [comment]
Those settings should work now: https://github.com/ImranR98/Obtainium/releases/tag/v1.1.4

**Comment by topcaser** [comment]
Thanks for looking into it. Unfortunately I tried this link to add ghostcommander again (I removed it from obtainium but didn't deinstall): https://sourceforge.net/projects/ghostcommander/files/Releases/

But obtainium says: no Version found...

**Comment by topcaser** [comment]
![Screenshot_20240417-035021](https://github.com/ImranR98/Obtainium/assets/10373812/957eeb9c-5427-4538-a4c0-b8b2ed7765db)

If I click on your link, the json is imported to obtainium. Yesterday I managed to install a version but it was a beta version and obtainium told me that a new version is available at once. The version numbering was lower than that number from the installed beta. Anyway I got an error message during install process. Therefore I deinstalled again and removed the app from obtainium again and utilized your link. But now I get a failure message again during install (see the screenshot). This is really hard stuff...
